ex3-solution/src/components/products/products.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+
+import { Products, ProductsItem } from "./";
+
+export default {
+  title: "Organisms/Products",
+  component: Products,
+};
+
+const Template = (args) => (
+  <Products>
+    {[...Array(args.items)].map(() => (
```

Comment:
i was looking at this for a minute until i realized that items is a number. probably naming it `count` is better considering you have `items` in quicklinks and there it's an array